src/QuadForm/Quad/ZGenus.jl

```diff
@@ -2587,12 +2706,31 @@ julia> NS = direct_sum(integer_lattice(:U), rescale(root_lattice(:A, 16), -1))[1
 julia> LK3, iNS, i = embed_in_unimodular(NS, 3, 19);
 
 julia> genus(LK3)
-ZZGenus
-Signature: (3, 19)
-Genus symbol at 2:   1^22
+Genus symbol for integer lattices
+Signatures: (3, 0, 19)
+Local symbol:
```

Comment:
It seems nothing is printed for the local symbol?

Reply:
Ah yes indeed, I will have a look at it.. thanks!